src/test/kotlin/pl/touk/krush/BookTest.kt

```diff
@@ -44,4 +47,11 @@ class BookTest {
         }
     }
 
+    /**
```

Comment:
\[detekt\] INFO: \[style/NamingConventionViolation\] All names in the codebase should be matching the naming convention of the codebase\.